pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -125,7 +386,7 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 
 		for i := 0; i < pages; i++ {
 			results := searchData.GetSearch()
-			searchRepos = append(searchRepos, results.Edges...)
+			searchRepos = append(searchRepos, results.Nodes...)
```

Comment:
Did we rename the Genqlient value from edges to nodes?

Reply:
Yep